sys/shell/src/shell_nlip.c

```diff
@@ -156,6 +156,10 @@ shell_nlip_mtx(struct os_mbuf *m)
     nwritten = 0;
     off = 0;
 
+    rc = console_lock(OS_TICKS_PER_SEC);
+    if (rc != OS_OK) {
+        goto err;
+    }
     /* Start a packet */
     console_write(pkt_seq, sizeof(pkt_seq));
```

Comment:
These calls to `console_write()` lock the mutex a second time.  This is legal, but I find it a bit confusing.  I don't know if it is worth adding a `console_write_nolock()` function, though.